### ISSUE TITLE
Issue with Pylance/autocompletion in new interactive window

### ISSUE THREAD

**Issue by StevenLi-DS** [microsoft/vscode-jupyter#7406]
```
Version: 1.60.0 (user setup)
Commit: e7d7e9a9348e6a8cc8c03f877d39cb72e5dfb1ff
Date: 2021-09-01T10:41:52.311Z
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043
Extension: v2021.9.0
```

1. `%config Completer.use_jedi = False` is no longer working, so no Jupyter autocompletion is enabled and no autocompletion for column names when using dot notation. 
https://github.com/microsoft/pylance-release/issues/851
https://github.com/microsoft/vscode-jupyter/issues/1300
![6k5EIihGuE](https://user-images.githubusercontent.com/33796896/131944413-e06720a0-009c-452c-9cce-a54ca4bbf731.gif)

1. The code entering field is fixed at the bottom, so the autocompletion blocks your code completely sometimes.
![Snipaste_220](https://user-images.githubusercontent.com/33796896/131944570-894d6ae4-8165-4b83-8e47-e89a7257435f.png)
![Snipaste_294](https://user-images.githubusercontent.com/33796896/131944572-976f14de-561b-4781-928d-5c7e14e87e9b.png)

1. unnecessary checking
![image](https://user-images.githubusercontent.com/33796896/131945129-4501b26b-b659-4f96-b981-b2930bcdeba2.png)

**Comment by claudiaregio** [comment]
Thanks @StevenLi-DS, will discuss at next triage meeting

**Comment by StevenLi-DS** [comment]
May I know why `%config Completer.use_jedi = False` is no longer working? I thought it is just Jupyter notebook settings.

**Comment by rchiodo** [comment]
This looks to be a bug in our autocomplete for the interactive window. We aren't asking jupyter for any autocomplete in the input box anymore.

**Comment by StevenLi-DS** [comment]
Is there a current workaround for the second issue I mentioned above?

> The code entering field is fixed at the bottom, so the autocompletion blocks your code completely sometimes.

**Comment by rchiodo** [comment]
> The code entering field is fixed at the bottom, so the autocompletion blocks your code completely sometimes.

Sorry not sure what you're referring to when you say it blocks your code? You mean it comes up on top of it visually? AFAIK this is by design

**Comment by StevenLi-DS** [comment]
> You mean it comes up on top of it visually?

Yes. I think the current design didn't take consideration of the fact that the code entering field is fixed in the lower right corner. It seems like this is a general issue in vscode if you type code in the lower right position. 
![TBiSMCWVMm](https://user-images.githubusercontent.com/33796896/134402255-be364422-899f-4898-a6eb-b0337f5436dd.gif)
![FAlQQVZS61](https://user-images.githubusercontent.com/33796896/134402257-602844e8-4fcd-4e28-a791-57fe90677193.gif)

**Comment by rchiodo** [comment]
Thanks. I created a new issue in VS code for that:
https://github.com/microsoft/vscode/issues/133622

**Comment by rchiodo** [comment]
Other than in this issue, I don't think we have a separate bug for it. Looks like there's a newline problem in the concatenation of the cells as we pass them to pylance.
